ada_feeding/ada_feeding/trees/move_to_configuration_tree.py

```diff
@@ -99,143 +96,50 @@ def __init__(
-        )
-        max_acceleration_scaling_factor_key = Blackboard.separator.join(
-            [move_to_namespace_prefix, "max_acceleration_scaling_factor"]
-        )
-        self.blackboard.register_key(
-            key=max_acceleration_scaling_factor_key,
-            access=py_trees.common.Access.WRITE,
-        )
-
-        # Write the inputs to MoveToConfiguration to blackboard
-        set_to_blackboard(
-            self.blackboard,
-            joint_positions_key,
-            self.joint_positions,
-            self.keys_to_not_write_to_blackboard,
-        )
-        set_to_blackboard(
-            self.blackboard,
-            tolerance_key,
-            self.tolerance,
-            self.keys_to_not_write_to_blackboard,
-        )
-        set_to_blackboard(
-            self.blackboard,
-            weight_key,
-            self.weight,
-            self.keys_to_not_write_to_blackboard,
-        )
-        set_to_blackboard(
-            self.blackboard,
-            pipeline_id_key,
-            self.pipeline_id,
-            self.keys_to_not_write_to_blackboard,
-        )
-        set_to_blackboard(
-            self.blackboard,
-            planner_id_key,
-            self.planner_id,
-            self.keys_to_not_write_to_blackboard,
-        )
-        set_to_blackboard(
-            self.blackboard,
-            allowed_planning_time_key,
-            self.allowed_planning_time,
-            self.keys_to_not_write_to_blackboard,
-        )
-        set_to_blackboard(
-            self.blackboard,
-            max_velocity_scaling_factor_key,
-            self.max_velocity_scaling_factor,
-            self.keys_to_not_write_to_blackboard,
-        )
-        set_to_blackboard(
-            self.blackboard,
-            max_acceleration_scaling_factor_key,
-            self.max_acceleration_scaling_factor,
-            self.keys_to_not_write_to_blackboard,
-        )
-
-        # Create the MoveTo behavior
-        move_to_name = Blackboard.separator.join([name, move_to_namespace_prefix])
-        move_to = MoveTo(move_to_name, tree_root_name, self._node)
-
-        # Add the joint goal constraint to the MoveTo behavior
-        joint_goal_constaint_name = Blackboard.separator.join(
-            [name, joint_constraint_namespace_prefix]
-        )
-        joint_constraints = SetJointGoalConstraint(
-            joint_goal_constaint_name, move_to, self._node
-        )
-
-        # Clear the constraints
-        if self.clear_constraints:
-            clear_constraints_name = Blackboard.separator.join(
-                [name, clear_constraints_namespace_prefix]
-            )
-            root = ClearConstraints(
-                clear_constraints_name, joint_constraints, self._node
-            )
-        else:
-            root = joint_constraints
-
-        tree = py_trees.trees.BehaviourTree(root)
-        return tree
+        # Root Sequence
```

Comment:
Just checking: is this a necessary tree given MoveToConfiguration w/ FT Thresholds exists? I guess it's not a big deal.

Reply:
Done, I deleted it. In practice none of our motions will use it.